### PR TITLE
Add CCMS EBS System Context diagram

### PR DESCRIPTION
**Pull request by bleach**
## What

I started describing CCMS by drawing the system context for eBusiness Suite (EBS).

The responsibilities of the main parts of CCMS (EBS, Provider UI, and Financials) are distinct enough to separate them out even at this level of detail.

This lists the main collaborators with CCMS EBS, but excluding SOA and Hub for clarity (they don't add a huge amount at this level of detail. I don't know enough about EDRM to include it yet.

## Suggested review feedback

- Check the names & relationships match reality
- Feedback on clarity
- Bonus: provide additional context on EDRM
- Bonus: check the yaml I've committed generates the diagrams I've committed

### CODE REVIEW

File: diagrams/ccms-ebs-system-context.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  name: LAA Benefit Checker
+  description: Confirms if individual is in receipt of a passported benefit by calling DWP benefit checker service
+  position: '1900,1200'
+
+relationships:
+- source: CCMS Provider UI
+  description: Submits form results via SOA
+  destination: CCMS eBusiness Suite
+- source: CCMS eBusiness Suite
+  destination: CCMS Financials
```
Comment (sldblog) [inline diagrams/ccms-ebs-system-context.yaml]: 🙋‍♂️ There's a missing `description` here. Is that intentional?

**Comment by sldblog** [comment]
😍 for this diagram!

**Comment by bleach** [comment]
I tried to follow the instructions at https://git-lfs.github.com/ before raising this PR by installing git-lfs from homebrew and then committing as normal, so I'm unclear what went wrong. 

I'll try again using a fresh clone and branch.

**Comment by bleach** [comment]
I've made sure that git lfs is being used now (unsure if it was a fresh clone or running `git lfs install` that fixed it).

The offending commit is still in the repo and according to [this stackoverflow answer](https://stackoverflow.com/questions/3765234/listing-and-deleting-git-commits-that-are-under-no-branch-dangling) will be removed automatically when the reflogs expire, so I propose letting that happen.

**Comment by bleach** [comment]
outstanding things:

- [ ] add the missing description on the link between EBS and Financials
- [ ] review Neal's feedback on slack and amend

File: diagrams/ccms-ebs-system-context.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+links:
+  The FC4 Framework: https://fundingcircle.github.io/fc4-framework/
+  Structurizr Express: https://structurizr.com/express
+---
+type: System Context
+scope: CCMS eBusiness Suite
+description: The system context diagram for CCMS eBusiness Suite. Deliberately omits communication details such as SOA and Information HUB
```
Comment (sldblog) [inline diagrams/ccms-ebs-system-context.yaml]: Could we expand somewhere on what "CCMS eBusiness Suite" is doing from a responsibility perspective, please?

**Comment by sldblog** [comment]
@ministryofjustice/laa-technical-architects Shall we merge this pull request as-is, or move the files to a service area folder first?

**Comment by sldblog** [comment]
I'll merge because this is useful and we can clean up as we go.

**Comment by sldblog** [comment]
🎩 🙇

**Pull request merged**